proximo/proximo_source.go

```diff
@@ -91,10 +95,11 @@ func (cm *consMsg) getMsgID() string {
 }
 
 func (ams *asyncMessageSource) ConsumeMessages(ctx context.Context, messages chan<- substrate.Message, acks <-chan substrate.Message) error {
-	rg, ctx := rungroup.New(ctx)
+	authCtx := setupAuthentication(ctx, ams.credentials)
+	rg, rgctx := rungroup.New(authCtx)
```

Comment:
We only need one context, so I think it's nicer to just keep this similar to the previous version.
```suggestion
	rg, ctx := rungroup.New(setupAuthentication(ctx, ams.credentials))
```